.gitignore

```diff
@@ -67,6 +67,7 @@ ipch/
 *.psess
 *.vsp
 *.vspx
+.vs
```

Comment:
would you mind adding `.out` as well? that is the default output directory of visual studio if it builds the cmake for you.

Reply:
It is probably `out`, not `.out`
I'll add it.

---
In reply to: [338160314](https://github.com/Azure/azure-ulib-c/pull/40#discussion_r338160314) [](ancestors = 338160314)

Reply:
ah sorry you are right. thanks

---
In reply to: [338164036](https://github.com/Azure/azure-ulib-c/pull/40#discussion_r338164036) [](ancestors = 338164036,338160314)